### ISSUE TITLE
Error on cargo install

### ISSUE THREAD

**Issue by SeseMueller** [tramhao/termusic#225]
I wanted to install termusic with cargo and ran 
```bash
cargo install termusic termusic-server
```
as described in the README.md. 
Both times, I got the error:
```rust
error[E0061]: this function takes 2 arguments but 1 argument was supplied
   --> /Users/USERNAME/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tui-realm-stdlib-1.3.1/src/components/table.rs:351:29
    |
351 |             let mut table = TuiTable::new(rows)
    |                             ^^^^^^^^^^^^^------ an argument is missing
    |
note: associated function defined here
   --> /Users/USERNAME/.cargo/registry/src/index.crates.io-6f17d22bba15001f/ratatui-0.26.1/src/widgets/table/table.rs:263:12
    |
263 |     pub fn new<R, C>(rows: R, widths: C) -> Self
    |            ^^^
help: provide the argument
    |
351 |             let mut table = TuiTable::new(rows, /* widths */)
    |                                          ~~~~~~~~~~~~~~~~~~~~

error[E0277]: the trait bound `tuirealm::tui::text::Text<'_>: From<&std::string::String>` is not satisfied
   --> /Users/USERNAME/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tui-realm-stdlib-1.3.1/src/components/table.rs:373:48
    |
373 |                 table = table.highlight_symbol(hg_str);
    |                               ---------------- ^^^^^^ the trait `From<&std::string::String>` is not implemented for `tuirealm::tui::text::Text<'_>`
    |                               |
    |                               required by a bound introduced by this call
    |
    = help: the following other types implement trait `From<T>`:
              <tuirealm::tui::text::Text<'a> as From<&'a Masked<'_>>>
              <tuirealm::tui::text::Text<'a> as From<Masked<'a>>>
              <tuirealm::tui::text::Text<'a> as From<std::string::String>>
              <tuirealm::tui::text::Text<'a> as From<&'a str>>
              <tuirealm::tui::text::Text<'a> as From<Cow<'a, str>>>
              <tuirealm::tui::text::Text<'a> as From<tuirealm::tui::text::Span<'a>>>
              <tuirealm::tui::text::Text<'a> as From<tuirealm::tui::text::Line<'a>>>
              <tuirealm::tui::text::Text<'a> as From<std::vec::Vec<tuirealm::tui::text::Line<'a>>>>
    = note: required for `&std::string::String` to implement `Into<tuirealm::tui::text::Text<'_>>`
note: required by a bound in `tuirealm::tui::widgets::Table::<'a>::highlight_symbol`
   --> /Users/USERNAME/.cargo/registry/src/index.crates.io-6f17d22bba15001f/ratatui-0.26.1/src/widgets/table/table.rs:501:32
    |
501 |     pub fn highlight_symbol<T: Into<Text<'a>>>(mut self, highlight_symbol: T) -> Self {
    |                                ^^^^^^^^^^^^^^ required by this bound in `Table::<'a>::highlight_symbol`
```

I am on stable cargo 1.76, rustc 1.76. 
I have an up-to-date M1 Macbook Air.

**Comment by SeseMueller** [comment]
Note that the installation instructions from git did work after `brew install`-ing gstream.
This is likely because the cargo install wants to use tui-realm-stdlib v1.3.1 and the git install v1.2.0.

**Comment by tramhao** [comment]
You should be able to install with:
```
cargo install termusic termusic-server --locked
```
Please confirm so that I could update README. Thanks.

**Comment by SeseMueller** [comment]
Can confirm, works just as expected. 
(Even the config is kept when switching source)
Thanks!

**Comment by tramhao** [comment]
fixed in ac006a8c34763b7181371e948109964c4fcc5340
